pkg/i18n/english.go

```diff
@@ -1089,12 +1098,17 @@ func EnglishTranslationSet() *TranslationSet {
 		NewBranchNameBranchOff:               "New branch name (branch is off of '{{.branchName}}')",
 		CantDeleteCheckOutBranch:             "You cannot delete the checked out branch!",
 		DeleteBranchTitle:                    "Delete branch '{{.selectedBranchName}}'?",
+		DeleteBranchesTitle:                  "Delete selected branches?",
 		DeleteLocalBranch:                    "Delete local branch",
+		DeleteLocalBranches:                  "Delete local branches",
 		DeleteRemoteBranchOption:             "Delete remote branch",
 		DeleteRemoteBranchPrompt:             "Are you sure you want to delete the remote branch '{{.selectedBranchName}}' from '{{.upstream}}'?",
+		DeleteRemoteBranchesPrompt:           "Are you sure you want to delete the remote branches of the selected branches from their respective remotes?",
```

Comment:
Adding 'from their respective remotes' feels superfluous (likewise in DeleteLocalAndRemoteBranchesPrompt). Thoughts?

Reply:
I thought it be good to be explicit about the fact that it actually goes to the remote and deletes the branch there (as opposed to only deleting the tracking branch from the local repo, maybe). We are also very explicit about this when deleting a single branch, this is just analogous to that.

Reply:
Makes sense